hw/block/ox-ctrl/mmgr/volt/volt.c

```diff
@@ -58,9 +58,10 @@ static uint32_t volt_get_next_prp(struct volt_dma *dma, uint32_t ch){
             usleep(1);
             continue;
         }
-        pthread_mutex_unlock(&prpmap_mutex[ch]);
+        
+        prp_map[ch] |= prp_map[ch] | (1 << (next - 1));
```

Comment:
Did you change this line? There is a redundant | there...
It should be
prp_map[ch] |= 1 << (next - 1);
or
prp_map[ch] = prp_map[ch] | (1 << (next - 1));